Add documentation for contriburors (#484)
cloudposse/docs: content/community/contributors.md

consulting engagements with our customers. Everything we do is Open Sourced under our GitHub under the permissive APACHE2 license. With so many projects, however, it wouldn't be possible to maintain all of them if we didn't have the 
support from our community and some tools to make life easier.

## Our Tools

Here are some of the tools we depend on for running our Open Source organization.

| Tool            | Description                                                                                                                    |
| --------------- | ------------------------------------------------------------------------------------------------------------------------------ |
| `geodesic`      | [Cloud Automation Shell](https://github.com/cloudposse/geodesic) that we use as the base Docker image for many of our projects |
| `build-harness` | Our collection of GNU-style `Makefiles` for building stuff                                                                     |
| `README.yaml`   | Our specification for generating beautiful READMEs                                                                             |

> | `packages`      | Our toolchain  that we rely on throughout our projects                                                                         |

Reviewer: ```suggestion
| `packages`      | Our toolchain that we rely on throughout our projects                                                                         |
```

| GitHub Actions  | Our CI/CD Platform                                                                                                             |

## How to Become a Contributor

> To become a contributor is easy. Just start opening Pull Requests with enhancements, bugfixes, or other improvements. 

Reviewer: ```suggestion
Becoming a contributor is easy. Just start opening Pull Requests with enhancements, bugfixes, or other improvements. 
```

| -------------------------------------------------------------------------------- | ------------------------------------------- | ------------- |
| ![osterman](https://img.cloudposse.com/75x75/http://github.com/osterman.png)     | [@osterman](https://github.com/osterman)    | Erik Osterman |
| ![aknysh](https://img.cloudposse.com/75x75/http://github.com/aknysh.png)         | [@aknysh](https://github.com/aknysh)        | Andriy Knyshh |
| ![goruha](https://img.cloudposse.com/75x75/http://github.com/goruha.png)         | [@goruha](https://github.com/goruha)        | Igor Rodionov |
| ![nuru](https://img.cloudposse.com/75x75/http://github.com/osterman.png)         | [@nuru](https://github.com/nuru)            | Jeremy        |
| ![jamengual](https://img.cloudposse.com/75x75/http://github.com/jamengual.png)   | [@jamengual](https://github.com/jamengual)  | PePe Amengual |
| ![adamcrews](https://img.cloudposse.com/75x75/http://github.com/adamcrews.png)   | [@adamcrews](https://github.com/jamengual)  | Adam Crews    |
| ![nitrocode](https://img.cloudposse.com/75x75/http://github.com/nitrocode.png)   | [@nitrocode](https://github.com/nitrocode)  | Ronak         |

> | ![jamengual](https://img.cloudposse.com/75x75/http://github.com/jamengual.png)   | [@jamengual](https://github.com/jamengual)  | PePe Amengual |

Reviewer: already exists

> | ![RothAndrew](https://img.cloudposse.com/75x75/http://github.com/RothAndrew.png) | [@jamengual](https://github.com/RothAndrew) | Andrew Roth   |

Reviewer: ```suggestion
| ![RothAndrew](https://img.cloudposse.com/75x75/http://github.com/RothAndrew.png) | [@RothAndrew](https://github.com/RothAndrew) | Andrew Roth   |
```
